mint-auditor/migrations/2022-05-24-214504_create_initial/up.sql

```diff
@@ -0,0 +1,37 @@
+-- Audit data per block
+CREATE TABLE block_audit_data (
+    -- Diesel requires having a primary key and sqlite doesn't allow 64 bit primay keys, so even though
+    -- we would've wanted to use the block_index for that we can't.
+    id INT PRIMARY KEY,
+    block_index UNSIGNED BIGINT NOT NULL
+    -- Future revision would add gnosis safe data here
+);
+CREATE UNIQUE INDEX idx__block_audit_data__block_index ON block_audit_data(block_index);
+
+-- Balance per token, for each block we have audited.
+CREATE TABLE block_balance (
+    -- Diesel requires having a primary key and sqlite doesn't allow 64 bit primay keys, so even though
+    -- we would've wanted to use the block_index for that we can't.
+    id INT PRIMARY KEY,
+    block_index UNSIGNED BIGINT NOT NULL,
```

Comment:
Same comment as above with the following caveat:

If we're using `block_index` as primary key, that only allows one record. You wouldn't be able to record multiple `token_id`s. I don't think `block_index` would work in this table as the primary key.